cclient/BUILD

```diff
@@ -1,28 +1,28 @@
 cc_library(
     name = "api",
-    srcs = glob([
-        "*.c",
-        "**/*.c",
-        "**/*.h",
-    ]),
+    srcs = [
+        "iota_api.c",
+        "iota_api.h",
+    ],
     hdrs = ["iota_api.h"],
     visibility = ["//visibility:public"],
     deps = [
-        ":service",
+        "//cclient:service",
```

Comment:
why full path?

Reply:
I thought it's no difference in this case, isn't it?